sample-app/Katamari_Integration/trigger-oom.yml

```diff
@@ -0,0 +1,22 @@
+---
```

Comment:
I would think right now it's making more sense, because these are different type of yml, tasks folders are yml for individul task only, what's outside of that folder are the playbooks.

Reply:
Agree, so, instead of putting all stuff in root level folder, these are playbooks deserved to be put in a separate sub-folder, and they will call those in tasks folder. Ultimately, only one or two shell script needed in root folder, which is the main entrance to call other ansible stuff.

Reply:
yeah, we can create a playbook folder for that.